tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1360,4 +1361,26 @@ public function commentsProvider() : array
             'valid with extra opening brackets' => ['(DC2Type:should((.stop)).before)', 'should((.stop', $currentType],
         ];
     }
+
+    public function testCreateAndListSequences() : void
+    {
+        if ( ! $this->_sm->getDatabasePlatform()->supportsSequences()) {
+            $this->markTestSkipped('This test is only supported on platforms that support sequences.');
+        }
+
+        $sequence1Name = 'sequence_1';
```

Comment:
This seems to error on Oracle now:
```
1) Doctrine\Tests\DBAL\Functional\Schema\OracleSchemaManagerTest::testCreateAndListSequences 
Exception: [PHPUnit\Framework\Error\Notice] Undefined index: sequence_1 
```
https://continuousphp.com/git-hub/doctrine/dbal/build/c3aeacf9-fd40-45c2-8304-e93fc34a5bee/output

Reply:
Yup, have to setup oracle here to reproduce.

Reply:
Most likely, it’s upper case on Oracle.

Reply:
Indeed, it is